doc/controls/CardAndCardContentControl.md

```diff
@@ -0,0 +1,120 @@
+# Card & CardContentControl
+
+## Summary
+
+Card represents a control identifiable as a single, contained unit used to visually group related child content and actions that relate information about a subject.
+A card’s layout and dimensions depend on its contents.
```

Comment:
maybe move this line under ##card?

Reply:
I decided to change the content to do a summary for both at the same time as they have the same definition at the end